Log source origin for Journald integration (#7683)
DataDog/documentation: content/en/integrations/journald.md

      container_mode: true
```

> To fill `source` and `service` attributes, the Agent collects `SYSLOG_IDENTIFIER` , `_SYSTEMD_UNIT` and `_COMM`and set them to the first non empty value. In order to take advantage of the integration pipelines, Datadog recommends to set the `SYSLOG_IDENTIFIER` parameter of the systemd service configuration file found in `/etc/systemd/system/xxx.service`

Reviewer: note that the path to the service file (unit file) depends on the distribution. `systemctl show -p FragmentPath <unit_name>` allows knowing where the service file is, we could document it here.

Users will want to set `SyslogIdentifier` (https://www.freedesktop.org/software/systemd/man/systemd.exec.html#SyslogIdentifier=) in the unit file directly, or in an override file (see https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Description), might be worth documenting.

```suggestion
To fill `source` and `service` attributes, the Agent collects `SYSLOG_IDENTIFIER` , `_SYSTEMD_UNIT` and `_COMM`and set them to the first non empty value. In order to take advantage of the integration pipelines, Datadog recommends to set the `SyslogIdentifier` parameter on the systemd service file directly, or in a systemd service override file. Their location depends on your distribution, but you can find the location of the systemd service file with the command `systemctl show -p FragmentPath <unit_name>`.
```

> To fill `source` and `service` attributes, the Agent collects `SYSLOG_IDENTIFIER` , `_SYSTEMD_UNIT` and `_COMM`and set them to the first non empty value. In order to take advantage of the integration pipelines, Datadog recommends to set the `SyslogIdentifier` parameter on the systemd service file directly, or in a systemd service override file. Their location depends on your distribution, but you can find the location of the systemd service file with the command `systemctl show -p FragmentPath <unit_name>`.

Reviewer 2: ```suggestion
To fill `source` and `service` attributes, the Agent collects `SYSLOG_IDENTIFIER` , `_SYSTEMD_UNIT` and `_COMM`and set them to the first non empty value. In order to take advantage of the integration pipelines, Datadog recommends setting the `SyslogIdentifier` parameter in the `systemd` service file directly, or in a `systemd` service override file. Their location depends on your distribution, but you can find the location of the `systemd` service file by using the command `systemctl show -p FragmentPath <unit_name>`.
```

> To fill `source` and `service` attributes, the Agent collects `SYSLOG_IDENTIFIER` , `_SYSTEMD_UNIT` and `_COMM`and set them to the first non empty value. In order to take advantage of the integration pipelines, Datadog recommends to set the `SYSLOG_IDENTIFIER` parameter of the systemd service configuration file found in `/etc/systemd/system/xxx.service`
> 
> **Note**: With Agent 7.17+ if `container_mode` is set to `true`, the previous default behaviour is overwritten. The `source` attribute of your logs is automatically set to the corresponding short image name of the container instead of simply `docker`.

Reviewer: let's mention that this applies to logs coming from containers only, otherwise I find this a bit hard to understand:

```suggestion
To fill `source` and `service` attributes, the Agent collects `SYSLOG_IDENTIFIER` , `_SYSTEMD_UNIT` and `_COMM`and set them to the first non empty value. In order to take advantage of the integration pipelines, Datadog recommends to set the `SYSLOG_IDENTIFIER` parameter of the systemd service configuration file found in `/etc/systemd/system/xxx.service`

**Note**: With Agent 7.17+ if `container_mode` is set to `true`, the default behavior changes for logs coming from docker containers. The `source` attribute of your logs is automatically set to the corresponding short image name of the container instead of simply `docker`.
```

Author: updated! thanks!

> **Note**: With Agent 7.17+ if `container_mode` is set to `true`, the default behavior changes for logs coming from docker containers. The `source` attribute of your logs is automatically set to the corresponding short image name of the container instead of simply `docker`.

Reviewer 2: ```suggestion
**Note**: With Agent 7.17+, if `container_mode` is set to `true`, the default behavior changes for logs coming from docker containers. The `source` attribute of your logs is automatically set to the corresponding short image name of the container instead of simply `docker`.
```
